python/private/pypi/attrs.bzl

```diff
@@ -15,6 +15,12 @@
 "common attributes for whl_library and pip_repository"
 
 ATTRS = {
+    "add_libdir_to_library_search_path": attr.bool(
```

Comment:
NOTE: this is enabled for both, WORKSPACE and bzlmod: https://rules-python--2476.org.readthedocs.build/en/2476/api/rules_python/python/extensions/pip.html#pip.parse.add_libdir_to_library_search_path

However, bzlmod will not forward this value to underlying `whl_library` instances, so `extension.bzl` needs to be updated. I think we should only forward this value to `whl_libraries` that are following the legacy `pip` way of downloading deps or downloading an sdist.